README.md

```diff
@@ -255,6 +245,79 @@ Check what the current default is in the buildout recipe, and update it:
 - Issue Tracker: https://github.com/collective/pas.plugins.oidc/issues
 - Source Code: https://github.com/collective/pas.plugins.oidc
 
+### Local Development Setup
+
+You need a working `python` environment (system, `virtualenv`, `pyenv`, etc) version 3.8 or superior.
+
+Then install the dependencies and a development instance using:
```

Comment:
I am not (yet) generally using pip. For me `make install` was not enough, I also had to do:

```
python3.11 -m venv .
bin/pip install cookiecutter mxdev -c contraints.txt
```

Have I made a mistake or forgotten something?